### ISSUE TITLE
Decreased plasma apolipoprotein C-II 

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7037]
**Preferred term label:**
Decreased circulating apolipoprotein C-II concentration

**Synonyms**
Decreased plasma apolipoprotein C-II 

**Definition (free text, please give PubMed ID)**
Reduced concentration of apolipoprotein C-II in the blood circulation.

**Parent term (use hpo.jax.org/app)**
Decreased circulating apolipoprotein concentration HP:0033459

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM:207750 
PMID: 6547689 2/2 and PMID: 213719 7/7 PMID: 3944267 2/2
**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
